bfabric/src/bfabric/entities/core/entity.py

```diff
@@ -27,6 +28,14 @@ def id(self) -> int:
         """Returns the entity's ID."""
         return int(self.__data_dict["id"])
 
+    @property
+    def uri(self) -> EntityUri:
+        """Returns the entity's URI."""
+        if self._client is None:
```

Comment:
This does not solve the original problem yet of requiring the client object to infer the URI